docs/docs/features/studio.md

```diff
@@ -10,6 +10,37 @@ ZMK Studio is in beta. Although every effort has been made to provide a stable e
 
 ZMK Studio provides runtime update functionality to ZMK powered devices, allowing users to change their keymap layers without flashing new firmware to their keyboards. Studio is still under active development, and is not yet ready for casual end user use.
 
+## Capabilities
+
+ZMK Studio currently has the following capabilities:
+
+| Legend: | ✅ Supported | 🚧 Under Development | 💡 Planned | 🕯️ Low Priority | ❌ Not Planned |
+| :------ | :----------- | :------------------- | :--------- | :-------------- | -------------- |
+
+| Studio Status | **Feature or Capability**                                                                         |
+| :-----------: | ------------------------------------------------------------------------------------------------- |
+|      ✅       | Making keymap changes while keyboard is in use                                                    |
+|      ✅       | Making changes while connected via USB                                                            |
+|      ✅       | Native app for: Windows, Linux, MacOS                                                             |
+|      ✅       | Making changes while connected via BLE (Linux web-app & native apps only)                         |
+|      ✅       | Assigning [predefined behaviors](../keymaps/behaviors/index.mdx) to keys                          |
+|      ✅       | Assigning [user-defined behaviors](../keymaps/behaviors/index.mdx#user-defined-behaviors) to keys |
+|      💡       | [Configuring predefined behaviors](../config/behaviors.md)                                        |
+|      💡       | [Configuring user-defined behaviors](../config/behaviors.md)                                      |
+|      💡       | [Configuring tap-dance](../keymaps/behaviors/tap-dance.mdx)                                       |
+|      💡       | [Configuring macros](../keymaps/behaviors/macros.md)                                              |
```

Comment:
I was a bit confused on why these were split from "user-defined behaviors" since they technically are. Is what we mean here is implementing a special UI to configure them?

If so, can they still be configured without a special UI when "configuring user-defined behaviors" is implemented? Or does it mean that former doesn't encapsulate these because e.g. `bindings` will not be editable in Studio?

Reply:
Macros have a very specific functionality that's going to require some careful work to make editing the `bindings` easy for end users. It will definitely be an extension/additional effort on top of the generic behavior property editing we'll have to build out.

Reply:
So should we qualify these as e.g. "Macro configuration interface", or does this mean we should edit the "configuring user-defined behaviors" item to exclude tap-dances+macros?

Reply:
Since this is purely informational, I'm not sure the damage in keeping it separated so users can find it easily?

Reply:
Adjusted based on what I *think* the two of you agreed on on Discord.